admin/src/app/app.component.ts

```diff
@@ -1,11 +1,23 @@
-import {Component, OnInit} from '@angular/core';
-import {StateService} from '../core/services/state.service';
-import {Observable} from 'rxjs';
+import { AsyncPipe, NgIf } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { Observable } from 'rxjs';
+import { StateService } from '../core/services/state.service';
```

Comment:
Can we use imports via @ instead of these relative paths?

Reply:
Should be good now, thanks Basil.